README.md

```diff
@@ -1,2 +1,13 @@
 # Steam desktop files updater
 This script creates desktop entries for Steam games
+
+## Usage
+
+The script works by passing the steam `*.desktop` files path, as:
```

Comment:
Hmm, the positional argument is not a "*.desktop files path" but a "steam root" or whatever the Steam Client calls it. Defaults to `~/.local/share/Steam`.